dbt_semantic_interfaces/implementations/metric.py

```diff
@@ -182,13 +182,12 @@ class PydanticMetricAggregationParams(HashableBaseModel):
 
     semantic_model: str
 
-    # TODO SL-4116: make sure we recreate/reuse all the validations for measures
-    # for these fields, too.
+    # If you add fields to this, please make sure to update the transformation
```

Comment:
Wonder if we could have a test for this? i.e., dynamically loops thru all the fields on this object and makes sure that `to_metric_aggregation_params()` populates them to be non-empty?
Not important if this is annoying, just an idea you can feel free to ignore 🙃